src/SmartComponents/SystemPackages/__snapshots__/SystemPackages.test.js.snap

```diff
@@ -6408,7 +6414,7 @@ exports[`SystemPackages.js Should match the snapshots 1`] = `
                                       getMenuRef={null}
                                       id="pagination-options-menu-bottom-toggle-1"
                                       isActive={false}
-                                      isDisabled={true}
+                                      isDisabled={0}
```

Comment:
This looks a bit fishy to me. Is that right?

Reply:
I updated the snapshots. Maybe it is due to some changes in the dependencies. But I would not care much about this :)

Reply:
OK. :D